test/extended/networking/network_segmentation.go

```diff
@@ -37,6 +37,9 @@ import (
 )
 
 const openDefaultPortsAnnotation = "k8s.ovn.org/open-default-ports"
+const pollTimeout = 2 * time.Minute
```

Comment:
make this a specific timeout related to the bug we are seeing with containers. Don't make a generic poll timeout that is used for getting a NAD or doing other things. Let's make it specific to this issue.

Reply:
what tim meant was to not change the "get NAD" eventually blocks to just let them use whatever number they had before...not a variable change.. so repushed to accommodate that suggestion